src/platform_impl/windows/window.rs

```diff
@@ -523,34 +518,17 @@ impl Window {
                             position.1,
                             size.0 as i32,
                             size.1 as i32,
-                            winuser::SWP_ASYNCWINDOWPOS | winuser::SWP_NOZORDER,
+                            winuser::SWP_NOOWNERZORDER | winuser::SWP_FRAMECHANGED,
```

Comment:
I'm not exactly sure what to put here, I just aligned it with https://devblogs.microsoft.com/oldnewthing/20100412-00/?p=14353.

Reply:
I think the previous flags make sense here, would prefer to keep them